x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/optimizer/Optimizer.java

```diff
@@ -1374,7 +1415,7 @@ private Expression literalToTheRight(BinaryOperator<?, ?, ?, ?> be) {
      * Propagate Equals to eliminate conjuncted Ranges.
      * When encountering a different Equals or non-containing {@link Range}, the conjunction becomes false.
      * When encountering a containing {@link Range}, the range gets eliminated by the equality.
-     * 
+     *
```

Comment:
I have my editor to trim trailing whitespaces from lines.

Reply:
Can you please change that to apply only to the modified lines and not the whole fail?
Thanks.
Regardless of the rules chosen, we tend to apply formatting once per project/packages otherwise just for the stuff added.
This reduces the noise when multiple folks work on the same file (like this case).